tests/module_loading/module_negative_path/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Using the default setting such that the library will be generated
```

Comment:
hmm.. I tried the option. It turns out that the copied library still has the same `SONAME` so that loader will look up the original library 😢 As a result, we still need to recompile the binary so the the name can be different.

Reply:
Are you trying to run the test against the same enclave as the success case? i.e. you're binplacing the dependency twice, but not primary enclave bin? It's not really important to optimize this though.

Reply:
nvm. Issue resolved 😄